icaruscode/CRT/CRTTPCTruthEffMCKalman_module.cc

```diff
@@ -0,0 +1,1721 @@
+    void getCatCrossXYZ(recob::Track trk, double &my_x, double &my_y, double &my_z);
+    void getTickRange(std::vector<art::Ptr<recob::Hit>> hits, int &mintick, int &maxtick);
+    void findIntersectingCRTVolume(TVector3 &retvec, TVector3 &endpoint, int &likely_CRT_region1, int &likely_CRT_region2, int &likely_CRT_region3);
+    void pointChecker(recob::Track trk, bool &startisinTPC, bool &endisinTPC, int &startflag, int &endflag);
+    void endpoint_PCA_ana(recob::Track trk, bool usestartpt, TVector3 &bestfit_dir, TVector3 &bestfit_pos);
+    float vect_dist(TVector3 vec1, TVector3 vec2);
+    bool willEnterCRTVolume(TVector3 vecpos, TVector3 vecdir, int crtregion);
+    double getnewDCA(TVector3 CRTHitpos, TVector3 TPC_posvec, TVector3 TPC_dirvec);
+    double CRT_plane_dist(TVector3 TPC_track_pos, TVector3 TPC_track_dir, TVector3 CRT_hit_pos, int plane_axis/*0 for x, 1 for y, 2 for z*/);
+
+
+  private:
+    art::ServiceHandle<art::TFileService> tfs;
+    icarus::crt::CRTBackTracker bt;
+
+    // Params got from fcl file.......
+    //    art::InputTag fTpcTrackModuleLabel; 	     	///< name of track producer
+    std::vector<art::InputTag> fTpcTrackModuleLabel; 	///< name of track producer
+    std::vector<art::InputTag> fPFParticleLabel;     	///< labels for source of PFParticle
+    std::vector<art::InputTag> fPFParticleMetadataLabel;///< label for PFParticleMetadata
+    art::InputTag              fCrtHitModuleLabel;   	///< name of crt producer
+    art::InputTag              fTriggerLabel;        	///< labels for trigger
+    art::InputTag 	       fSimModuleLabel;      	///< name of detsim producer
+    art::InputTag	       fMCParticleLabel;     	///<label for MCParticles
+    art::ServiceHandle<cheat::ParticleInventoryService > partInventory;
+    CRTT0MatchAlg              t0Alg;		     	///< used to call matching functions
+    bool                       fVerbose;       	     	///< print information about what's going on
+    bool		       fIsData;              	///< switch for if this is data or MC
+    bool	tpcEast, tpcWest;			///< true if in that TPC, false otherwise
+
+    geo::GeometryCore const* fGeometryService;   ///< pointer to Geometry provider
+    icarus::crt::CRTCommonUtils* fCrtutils;
+    //  CRTCommonUtils* fCrtutils;
+
+    TTree* tr_crttpc;
+    int fEvent;        			///< number of the event being processed
+    int fRun;          			///< number of the run being processed
+    int fSubRun;       			///< number of the sub-run being processed
+    int crt_region;    			///< CRT hit region code
+    int ttl_tpctrks, ttl_crthits; 	///< Total number of TPC tracks and CRT Hits for a given event
+    int driftdir;			///< Drift direction of the track
+    int num_crt_candidates;		///< Number of CRT Hit candidates for a given track, including the matched hti
+	    int startflag, endflag;		///< flags for TPC track endpoints
```

Comment:
If there isn't a good reason for the sudden indentation here, I would try to keep the indentation consistent.